apis/ec2/v1alpha1/vpc_types.go

```diff
@@ -0,0 +1,150 @@
+/*
```

Comment:
We should probably name this file `vpccidrblock_types.go` instead of `vpc_types.go`